fastlane/metadata/android/nl/full_description.txt

```diff
@@ -0,0 +1,5 @@
+IRMA geeft jou de controle en houdt je persoonlijke data veilig.
+
+De IRMA app beheert jouw persoonlijke IRMA kaartjes: je kunt nieuwe kaartjes ontvangen, ze selectief vrijgeven aan anderen, en ze verbinden aan digitale handtekeningen. Voorbeelden van kaartjes zijn "ik ben ouder dan 18", "mijn naam is ..." en "ik heb toegang tot ...". Deze staan veilig op jouw telefoon opgeslagen, en worden beschermd door jouw PIN-code.
+
+IRMA wordt gemaakt en onderhouden door de <a href="https://privacybydesign.foundation/irma">Stichting Privacy by Design Foundation</a> in samenwerking met <a href="https://sidn.nl/">SIDN</a>, en is gratis en open-source. Lees meer op <a href="https://irma.app">irma.app</a>. Voor de IRMA broncode, zie <a href="https://github.com/privacybydesign">github.com/privacybydesign</a>.
```

Comment:
Looks like one paragraph from the EN description is missing here, most likely:

> Persoonlijke informatie die u in de app verzamelt, wordt uitsluitend op uw telefoon opgeslagen (in plaats van ook in de cloud). Zelfs de organisatie achter IRMA kan uw informatie niet zien.

Reply:
I have rewritten the English version to more closely match the one in Dutch, and removed this paragraph, because what it says is mostly already covered by the preceding paragraph.

Reply:
Hehe… well, I need to disclose I'm not a Dutch speaker. I can read it, and even vaguely understand it – but above translation comes from Dr. Deepl :speak_no_evil:

So which way ever you feel, it will be right. I just followed my instinct that said "it looks different"…